### DISCUSSION TITLE
Use pyhydrogeo to get land use at specific lat / lon

### DISCUSSION THREAD

**Discussion by kovar-ursa** [hyriver/pygeohydro#79] [Q&A] [answered]
Good morning,

For an aviation use case - "Where did the drone launch?' - I'd like to get the land use for a lot of points in the U.S. 

I think I can use the NLCD to do this and the PyGeoHydro example seems to almost get me there. (https://hyriver.readthedocs.io/en/latest/notebooks/nlcd.html)

Observations:

1) I do not want to plot a lot of points, I want to get a text value for a specific point
2) I need to do this on an ongoing basis and I'd like to do this without hammering the servers

Questions:

1) If I set a very small bounding box - 10m for example - will I get the land use for just that point?
2) How do I extract the land use text value from the data that is provided by the server?

Thank you very much.

**Comment by cheginit** [comment, marked as answer]
@kovar-ursa Good morning!

Answer to your questions:

1. You will still get a dataset that you need to pass to `cover_statistics` function to get the summaries. Since resolution of NLCD is 30 m, it's best if you set the resolution to 30 m and use a bounding box with 30 m sides around the point of interest.
2. If by text you mean the land cover type you can do something like this:

```python
stats = gh.cover_statistics(lulc.cover_2019)
type_max = max(stats, key=stats.get)
```

However, the current implementation of `nlcd` function is not meant for point-based requests. I can see a couple of ways for implementing an efficient function for this specific purpose. Since this can be a useful feature for other users as well, please open a feature request for NLCD by location and I will work on it.

**Reply by kovar-ursa** [reply]
Thank you very much!
